pkg/apis/serving/register.go

```diff
@@ -27,6 +27,10 @@ const (
 	// generation of the Configuration that created this revision
 	ConfigurationGenerationAnnotationKey = GroupName + "/configurationGeneration"
 
+	// RevisionLastPinnedAnnotationKey is the annotation key used for determining when a route has
+	// pinned a revision
+	RevisionLastPinnedAnnotationKey = GroupName + "/lastPinned"
```

Comment:
Consider breaking these `pkg/apis/...` changes into a separate PR, since I think they are looking pretty good and don't want to hold them up.